ui/pages/notifications/notification-components/erc1155-sent-received/erc1155-sent-received.tsx

```diff
@@ -0,0 +1,208 @@
+import React from 'react';
+import { TRIGGER_TYPES } from '../../../../../app/scripts/controllers/metamask-notifications/constants/notification-schema';
+import { CHAIN_IDS } from '../../../../../shared/constants/network';
+import { t } from '../../../../../app/scripts/translate';
```

Comment:
I want to absolutely make sure this is okay to do.

Other react components import and use the hook, not the `t` function directly.